src/Workspaces/Remote/Core/AbstractAssetProvider.cs

```diff
@@ -24,17 +24,17 @@ internal abstract class AbstractAssetProvider
 
     public async Task<SolutionInfo> CreateSolutionInfoAsync(Checksum solutionChecksum, CancellationToken cancellationToken)
     {
-        var solutionCompilationChecksums = await GetAssetAsync<SolutionCompilationStateChecksums>(AssetHint.None, solutionChecksum, cancellationToken).ConfigureAwait(false);
-        var solutionChecksums = await GetAssetAsync<SolutionStateChecksums>(AssetHint.None, solutionCompilationChecksums.SolutionState, cancellationToken).ConfigureAwait(false);
+        var solutionCompilationChecksums = await GetAssetAsync<SolutionCompilationStateChecksums>(AssetHint.SolutionOnly, solutionChecksum, cancellationToken).ConfigureAwait(false);
+        var solutionChecksums = await GetAssetAsync<SolutionStateChecksums>(AssetHint.SolutionOnly, solutionCompilationChecksums.SolutionState, cancellationToken).ConfigureAwait(false);
```

Comment:
lots of the real product code now passes along it just wants to sync solution-level info.